content/pl/docs/setup/_index.md

```diff
@@ -0,0 +1,116 @@
+---
+reviewers:
+- brendandburns
+- erictune
+- mikedanese
+no_issue: true
+title: Od czego zacząć
+main_menu: true
+weight: 20
+content_template: templates/concept
+card:
+  name: setup
+  weight: 20
+  anchors:
+  - anchor: "#learning-environment"
+    title: Środowisko do nauki
+  - anchor: "#production-environment"
+    title: Środowisko produkcyjne  
+---
+
+{{% capture overview %}}
+
+Ten rozdział poświęcony jest różnym metodom konfiguracji i uruchomienia Kubernetes.
+
+Istnieje wiele rozwiązań dopasowanych do różnych potrzeb użytkowników: łatwości w utrzymaniu, wymagań bezpieczeństwa, poziomu sterowania, dostępności zasobów oraz niezbędnego doświadczenia do zarządzania klastrem.
+
+Klaster Kubernetes możesz zainstalować na lokalnym komputerze, w chmurze czy w prywatnym centrum obliczeniowym. Możesz też skorzystać z klastra Kubernetes udostępnianego jako usługa. Możesz też stworzyć swoje własne rozwiązanie w oparciu o różnych dostawców usług chmurowych, bądź bazując bezpośrednio na sprzęcie fizycznym.
```

Comment:
```suggestion
Klaster Kubernetes możesz zainstalować na lokalnym komputerze, w chmurze czy w prywatnym centrum obliczeniowym. Możesz też skorzystać z klastra Kubernetes udostępnianego jako usługa. Innym rozwiązaniem jest stworzenie własnego rozwiązania w oparciu o różnych dostawców usług chmurowych, bądź bazując bezpośrednio na sprzęcie fizycznym.
```

Reply:
I like it, but it has twice "rozwiązanie" - "rozwiązaniem jest stworzenie własnego rozwiązania"

Reply:
I would like to avoid is too much "możesz". Maybe:
```suggestion
Klaster Kubernetes możesz zainstalować na lokalnym komputerze, w chmurze czy w prywatnym centrum obliczeniowym. Możesz też skorzystać z klastra Kubernetes udostępnianego jako usługa. Inną możliwością jest własne rozwiązanie w oparciu o różnych dostawców usług chmurowych, bądź bazujące bezpośrednio na sprzęcie fizycznym.
```

Reply:
It's been rewritten, please have a look.